src/mmw/apps/modeling/geoprocessing.py

```diff
@@ -71,6 +72,16 @@ def sjs_submit(host, port, args, data, retry=None):
             except MaxRetriesExceededError:
                 raise Exception('No slots available in Spark JobServer.\n'
                                 'Details = {}'.format(response.text))
+        elif error['result'] == 'context geoprocessing not found':
+            geop_context_url = '{}/contexts?reset=reboot'.format(base_url)
+            context_response = requests.put(geop_context_url)
```

Comment:
An alternate possibility here would be to send this to Spark Job Server to create the "geoprocessing" context:

```py
geop_context_url = '{}/contexts/geoprocessing'.format(base_url)
context_response = requests.post(geop_context_url, data='')
```

Instead of resetting the contexts back to the initial config, this would create the "geoprocessing" context.